### PR TITLE
[WIP] fix speech demo

### PR DESCRIPTION
**Pull request by zhang3wind**

### CODE REVIEW

**Comment by piiswrong** [comment]
@antinucleon @pluskid

File: example/speech-demo/run_ami.sh [hunk 1]
```diff
@@ -13,7 +13,7 @@ set -u           #Fail on an undefined variable
 . ./cmd.sh
 . ./path.sh
 
-cmd=hostd3.pl
+cmd=run.pl
```
Comment (pluskid) [inline example/speech-demo/run_ami.sh]: Could you add a comment here to say that this is the kaldi wrapper to the distributed job scheduling system. To run locally, use `run.pl`, and refer to kaldi doc for other scheduling systems like sun grid engine and slurm.

**Comment by pluskid** [comment]
LGTM. @yzhang87 can you also take a look at this PR?

**Comment by yzhang87** [comment]
LGTM.

**Comment by antinucleon** [comment]
LGTM, but it is still in progress. We need to wait reproduce result.

**Pull request closed without merging**